lib/code-caller/index.js

```diff
@@ -28,27 +28,31 @@ const { PythonCaller } = require('./code-caller-python');
 let pool = null;
 
 module.exports = {
-  init() {
+  async init() {
     debug('init()');
 
     const { workersExecutionMode } = config;
     if (workersExecutionMode !== 'container' && workersExecutionMode !== 'native') {
       throw new Error(`unknown config.workersExecutionMode: ${workersExecutionMode}`);
     }
 
+    if (config.workersExecutionMode === 'container') {
+      await initCodeCallerDocker();
+    }
+
     const numWorkers = config.workersCount ?? Math.ceil(config.workersPerCpu * os.cpus().length);
     pool = genericPool.createPool(
       {
         create: async () => {
           let codeCallerOptions = {
             questionTimeoutMilliseconds: config.questionTimeoutMilliseconds,
-            errorLogger: logger.error,
+            errorLogger: logger.error.bind(logger),
```

Comment:
Grrrr: https://github.com/winstonjs/winston/blob/3998df01449da35a60cf4b90730b91f4b2466bc5/lib/winston/create-logger.js#L73

Reply:
Reason number 1000 to hate Winston 🤯